modules/gdscript/editor/script_templates/CharacterBody2D/basic_movement.gd

```diff
@@ -3,8 +3,9 @@
 extends _BASE_
 
 
-const SPEED = 300.0
-const JUMP_VELOCITY = -400.0
+const SPEED : float = 300.0
+const DECEL : float = 1000.0
```

Comment:
I think `DECEL` can be confusing to non-native English speakers, so it should be e.g. `DECELERATION`.

Reply:
Renamed